puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/containers.groovy

```diff
@@ -0,0 +1,26 @@
+import groovy.json.JsonOutput
+
+def createTimestamp() {
+    (new Date()).format("yyyyMMdd-HHmm", TimeZone.getTimeZone('UTC'))
+}
+
+def triggerGithubBuilder(project_name, version, version_build) {
+    def inputs = JsonOutput.toJson([
+        ref: 'master',
+        inputs: [
+            project_name: project_name,
+            version: version,
+            push: 'true',
+            registry: 'xyz',
+            version_build: version_build
+        ]
+    ])
+
+    def command_parts = [
+        "curl -X POST -H 'Accept: application/vnd.github.v3+json' -u $GITHUB_USER:$GITHUB_TOKEN",
+        "https://api.github.com/repos/xprazak2/image_builds/actions/workflows/build_image.yml/dispatches",
+        '-d \'' + inputs + '\''
```

Comment:
Perhaps we should use https://plugins.jenkins.io/http_request/ instead so there's no risk of an incorrectly escaped body?